tests/model_forms/tests.py

```diff
@@ -1789,6 +1717,49 @@ class WriterForm(forms.Form):
         self.assertEqual(len(params), 1)
 
 
+class MultipleChoiceWithFieldNameTests(TestCase):
```

Comment:
The class name seems a bit specific -- not sure any other tests would fit in this class (the existing organization is a bit poor).